manifests/apache/conf.pp

```diff
@@ -13,6 +13,7 @@
 # @param ldap_bind_authoritative Determines if other authentication providers are used when a user can be mapped to a DN but the server cannot bind with the credentials
 # @param ldap_require_group LDAP group to require on login
 # @param ldap_require_group_dn LDAP group DN for LDAP group
+# @param virtualenv_dir Set location where virtualenv will be installed
```

Comment:
thanks for the fix. can you also run `bundle exec rake reference` to update the REFERENCE.md?